README.Rmd

```diff
@@ -155,13 +156,35 @@ Core themes:
 
 Formatting functions:
 
+* `urbn_plot()`
+* `urbn_title()`
+* `urbn_subtitle()`
+* `urbn_y_title()`
+* `urbn_note()`
+* `urbn_source()`
 * `urbn_logo_text()`
-* `remove_ticks()`
-* `remove_axis()`
 * `scatter_grid()`
+* `remove_ticks()`
 * `add_axis()`
+* `remove_axis()`
+* `get_legend()`
+* `remove_legend()`
 * `urbn_geofacet`
 
+Palette functions:
+
+* `palette_urbn_main`
+* `palette_urbn_diverging`
+* `palette_urbn_quintile`
+* `palette_urbn_politics`
+* `palette_urbn_cyan`
+* `palette_urbn_gray`
+* `palette_urbn_yellow`
+* `palette_urbn_magenta`
+* `palette_urbn_green`
+* `palette_urbn_spacegray`
+* `palette_urbn_red`
```

Comment:
If these are all functions then I think there should be a `()` after each name to indicate each call.